src/java/org/apache/cassandra/io/util/FileHandle.java

```diff
@@ -498,7 +498,7 @@ public void close()
 
         private RebuffererFactory maybeCached(ChunkReader reader)
         {
-            if (chunkCache != null && chunkCache.capacity() > 0)
+            if (chunkCache != null && chunkCache.capacity() > 0)    //shall it be updated?
```

Comment:
In DSE this condition is shortened to `if (chunkCache != null)` but what is the point of caching an empty chunk?